parser/prism/Parser.h

```diff
@@ -65,13 +65,8 @@ class Parser final {
 
     static parser::ParseResult run(core::MutableContext ctx, bool directlyDesugar = true,
                                    bool preserveConcreteSyntax = false);
-    static ParseResult parseOnly(core::MutableContext &ctx);
-    static parser::ParseResult translateOnly(core::MutableContext &ctx, const Parser &parser, pm_node_t *node,
-                                             const std::vector<ParseError> &parseErrors,
-                                             const std::vector<core::LocOffsets> &commentLocations,
-                                             bool preserveConcreteSyntax);
 
-    ParseResult parse(bool collectComments = false);
+    ParseResult parseOnly(bool collectComments = false);
```

Comment:
I think we can document the reason for this function here. Highlight what the return type is and how it's different than `run()`